src/plugins/controls/public/options_list/components/options_list_strings.ts

```diff
@@ -126,7 +119,13 @@ export const OptionsListStrings = {
       }),
     getSortDisabledTooltip: () =>
       i18n.translate('controls.optionsList.popover.sortDisabledTooltip', {
-        defaultMessage: 'Ignore sorting when “Show only selected” is true.',
+        defaultMessage: 'Sorting is ignored when “Show only selected” is true',
+      }),
```

Comment:
@elastic/kibana-docs Since this tooltip is sticking around as described in the description of this PR, I reworded this to be passive rather than active. To me, the previous wording made it seem like the user could somehow change this behaviour, but they can't - whenever they click `"Show only selected"`, sorting is always ignored! 

![Dec-21-2022 16-28-01](https://user-images.githubusercontent.com/8698078/209022591-5ad07f81-6e2f-403b-98ce-af9c7257b2c0.gif)


Interested in your thoughts on this change 👍

Reply:
This change make sense to me.